packages/plugin-auto-tracking-browser/src/helpers.ts

```diff
@@ -51,3 +51,45 @@ export const getText = (element: Element): string => {
   }
   return text;
 };
+
+export const isPageUrlAllowed = (url: string, pageUrlAllowlist: string[] | undefined) => {
+  if (!pageUrlAllowlist || !pageUrlAllowlist.length) {
+    return true;
+  }
+  return pageUrlAllowlist.some((allowedUrl) => {
+    const allowedUrlRegex = new RegExp(allowedUrl);
```

Comment:
might be overkill, but a library like this one could provide really nice DX for defining path list

https://github.com/pillarjs/path-to-regexp

Reply:
Thanks for reviewing! The lib looks nice (thanks for sharing!), I don't think we need it at the moment, might be a little bit overkill.